go/connection.go

```diff
@@ -432,7 +433,7 @@ WAITING_FOR_RESULT:
 			obs.Scope().Timer(DriverName + ".query.StopQueryExecution").Record(timeStopQueryExecution)
 			obs.Log(ErrorLevel, "query canceled", zap.String("queryID", queryID))
 			return nil, ctx.Err()
-		case <-time.After(PoolInterval * time.Second):
+		case <-time.After(pollInterval):
```

Comment:
Can you add some unittest case for the new code/functions? Otherwise the code coverage will drop. Thank you. @keshav-dataco

Reply:
@henrywoo Done

Please review and merge if you are happy